.github/workflows/post_deploy_agent.yml

```
@@ -189,3 +189,21 @@ jobs:
         env:
           BUILD_PATH: ${{ github.workspace }}/build/NugetVersionDeprecator/NugetVersionDeprecator.csproj
           PUBLISH_PATH: ${{ github.workspace }}/publish
+
+  release-tags:
+    name: Create release tags
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create release tags for Lambda and K8s Init Containers
```

Please add the Harden Runner action: 
```suggestion
      - name: Harden Runner
        uses: step-security/harden-runner@0080882f6c36860b6ba35c610c98ce87d4e2f26f # v2.10.2
        with:
          disable-sudo: true
          egress-policy: audit
      - name: Create release tags for Lambda and K8s Init Containers
```